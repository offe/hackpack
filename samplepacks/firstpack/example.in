1
3
16
-2
