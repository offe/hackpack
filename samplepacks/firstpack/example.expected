1
9
256
4

